### PR TITLE
add redirects

### PR DESCRIPTION
**Pull request by renancaraujo**
Add redirects to the merch store and to discord.

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -56,12 +56,36 @@ const createCountdownPage = ({ createPage }) => {
...
   return Promise.all([
     createDocsPage({ createPage, graphql }),
     createFlameconPage({ createPage }),
     createCountdownPage({ createPage }),
+    createMerchPage({
+      createPage,
+    }),
+    createDiscordPage({
+      createPage,
+    }),
```
Comment (luanpotter) [inline gatsby-node.js:89]: we can inline these to follow the pattern above right?

File: src/templates/discordTemplate.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import createRedirect from "../components/redirect"
+export default createRedirect("https://discord.gg/pxrBmy4")
```
Comment (luanpotter) [inline src/templates/discordTemplate.js:2]: do we even need one file per each? can't we call createRedirect directly on gatsby-node.js ?

**Review by luanpotter — APPROVED** [review]
LGTM with a couple minor nits

File: src/templates/discordTemplate.js (hunk 2 above)
Comment (renancaraujo) [reply src/templates/discordTemplate.js:2]: Gatsby :(

**Pull request merged**